dask/dataframe/io/parquet/core.py

```diff
@@ -749,7 +749,7 @@ def create_metadata_file(
         )
     paths = sorted(paths, key=natural_sort_key)  # numeric rather than glob ordering
     ap_kwargs = {"root": root_dir} if root_dir else {}
-    root_dir, fns = _analyze_paths(paths, fs, **ap_kwargs)
+    paths, root_dir, fns = _sort_and_analyze_paths(paths, fs, **ap_kwargs)
```

Comment:
I take it we don't need the `paths = sorted(paths, key=natural_sort_key)` line here if the sorting will be done in `_sort_and_analyze_paths` anyway?

Reply:
yes, also an oversight, sorry about that, fixed